app/gui/view/component-browser/component-list-panel/src/button_panel.rs

```diff
@@ -0,0 +1,211 @@
+//! The [button panel](Model). This is a narrow bar on the top of the Component List Panel
+//! that contains various buttons for controlling the Component Browser.
+
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+
+use crate::grid::entry::icon;
+
+use enso_frp as frp;
```

Comment:
I believe that this is a part of prelude, so the import isn't necessary. Please also check other imports.